asv_bench/benchmarks/tslibs/normalize.py

```diff
@@ -30,6 +31,10 @@ def setup(self, size, tz):
         dti = pd.date_range("2016-01-01", periods=10, tz=tz).repeat(size // 10)
         self.i8data = dti.asi8
 
+        if size == 10 ** 6 and tz is tzlocal_obj:
```

Comment:
can we eliminate the size check (and to be honest do we even need this for *any* of them)

Reply:
> can we eliminate the size check

id rather not.  we went through a lot of trouble optimizing these and seeing the tradeoffs across different sizes

> and to be honest do we even need this for any of them

not sure what "this" is referring to

Reply:
meaning the 10 ** 6 size parameter here.

Reply:
again id rather not; it was a hassle checking/optimizing the perf across a range of sizes, would not want to lose that by accident